### PR TITLE
expand textboxes for fields in presets

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

**Comment by wardi** [comment]
Do you know if we're missing some classes on our input boxes to make them look like wet inputs?

**Comment by RabiaSajjad** [comment]
> Do you know if we're missing some classes on our input boxes to make them look like wet inputs?

I have applied the missing `form-control` class

File: ckanext/canada/internal/static/internal.css [hunk 1]
```diff
@@ -660,3 +660,22 @@ div.resource-upload-field .upload-type > label.control-label::before{
   color: #c6898b;
   margin-right: 2.5px;
 }
+.form-group .info-block {
+    padding: 0;
+    font-size: small;
+    color: #2572b4;
+}
+label.control-label:has(span.control-required):after {
+    content: "(required)";
```
Comment (wardi) [inline ckanext/canada/internal/static/internal.css:669]: does this need a translation?

File: ckanext/canada/templates/internal/base.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 {# Remove when WET v4.0.21 is out #}
 {%- block custom_styles %}
   {{ super() }}
-  <link rel="stylesheet" href="/internal.css?x=10" type="text/css" />
+  <link rel="stylesheet" href="/internal.css?x=11" type="text/css" />
```
Comment (wardi) [inline ckanext/canada/templates/internal/base.html:11]: If we used webassets for our css we wouldn't need this hack anymore

**Pull request merged**